tensorflow_probability/python/experimental/mcmc/BUILD

```diff
@@ -422,16 +423,19 @@ py_library(
     deps = [
         ":sample",
         ":sample_discarding_kernel",
+        ":tracing_reducer",
         ":with_reductions",
         # numpy dep,
         # tensorflow dep,
+        "//tensorflow_probability/python/mcmc/internal:util",
     ],
 )
 
 # py2and3
 py_test(
     name = "sample_fold_test",
     size = "small",
+    timeout = "long",
```

Comment:
Let's try to invest some energy in this not happening.

If the timeout is rising because of too many medium-size test methods adding up, increase the shard count instead.  If there's a single very long test method, see whether it can be avoided.  For instance, there's a `num_samples = 3000` at the end -- maybe don't run that one in all execution regimes?

(BTW, it's reasonable to have two test classes, one decorated with all execution regimes and one with something less profligate.)